tests/rip7560/testdata/prep.json

```diff
@@ -0,0 +1,14 @@
610132565b019061027b565b565b600080fd5b600080fd5b610eeb81610ca4565b03610ef257565b600080fd5b90503590610f0482610ee2565b565b600080fd5b67ffffffffffffffff8111610f2957610f256020916103dd565b0190565b6103e7565b90929192610f43610f3e82610f0b565b610555565b93818552602085019082840111610f5f57610f5d926106af565b565b610f06565b9080601f83011215610f8257816020610f7f93359101610f2e565b90565b6101da565b919091610180818403126110e157610fa0610180610555565b92610fae8160008401610ef7565b6000850152610fc081602084016101a4565b6020850152610fd281604084016101a4565b6040850152610fe481606084016101a4565b6060850152610ff681608084016101a4565b60808501526110088160a084016101a4565b60a085015261101a8160c084016101a4565b60c085015261102c8160e084016101a4565b60e085015261010082013567ffffffffffffffff81116110dc5781611052918401610f64565b61010085015261012082013567ffffffffffffffff81116110d75781611079918401610f64565b61012085015261014082013567ffffffffffffffff81116110d257816110a0918401610f64565b61014085015261016082013567ffffffffffffffff81116110cd576110c59201610f64565b610160830152565b610edd565b610edd565b610edd565b610edd565b610ed8565b9060208282031261111757600082013567ffffffffffffffff81116111125761110f9201610f87565b90565b61018a565b6100a6565b6111269051610ca4565b90565b611133905161012f565b90565b5190565b6111596111626020936111679361115081611136565b938480936106a6565b9586910161047e565b6103dd565b0190565b9a99989795939196949290966101808c019760008d0161118a91610cb0565b60208c0161119791610132565b60408b016111a491610132565b60608a016111b191610132565b608089016111be91610132565b60a088016111cb91610132565b60c087016111d891610132565b60e086016111e591610132565b8481036101008601526111f79161113a565b8381036101208501526112099161113a565b82810361014084015261121b9161113a565b9080820390610160015261122e9161113a565b90565b9061127b939291907fb80b6265e61634ae1f2f711376ce825a06f8655583a92b995c2d506a019524179161126f61126661009b565b92839283610eb4565b0390a1908101906110e6565b6112876000820161111c565b61129360208301611129565b906113306112a360408501611129565b936112b060608201611129565b906112bd60808201611129565b6112c960a08301611129565b6112d560c08401611129565b6112e160e08501611129565b906101008501519261012086015194610160610140880151970151977f0eaa55134db56fc26401bde6d5adf5b2691e06d153d390c39c73b766037c6f729c61132761009b565b9c8d9c8d61116b565b0390a1565b90565b63ffffffff60e01b1690565b60e01b90565b61135e61135961136392611335565b611344565b611338565b90565b61137363bf45c16661134a565b90565b90565b61138561138a91611338565b611376565b9052565b90565b61139d6113a29161012f565b61138e565b9052565b602093926113c66004836113be88956113ce97611379565b018092611391565b018092611391565b0190565b60200190565b6113e290516101b3565b90565b6113ff6113fa6113f483611136565b926113d2565b6113d8565b906020811061140d575b5090565b611420906000199060200360080261072d565b1638611409565b9061146861146d926114376108d8565b50611459611443611366565b919361144d61009b565b948593602085016113a6565b602082018103825203826103fd565b6113e5565b9056fea26469706673582212200e3d9e9d7dc1fb7fc07a8990305abbc5257c9219a49a46a30bc8af474e0e7b1b64736f6c63430008170033",
```

Comment:
What is this code? Is this needed? I would prefer it if we could avoid having a compiled solidity code checked into the repo for unit tests. We deploy contracts in Python integration tests and it is okay to have more things checked there IMO.